pkg/cmd/kind/load/image-archive/image-archive.go

```diff
@@ -78,19 +78,25 @@ func runE(logger log.Logger, flags *flagpole, args []string) error {
 		runtime.GetDefault(logger),
 	)
 
-	// Check if file exists
-	imageTarPath := args[0]
-	if _, err := os.Stat(imageTarPath); err != nil {
-		return err
+	for _, imageTarPath := range args {
+		if _, err := os.Stat(imageTarPath); err != nil {
+			return err
```

Comment:
I don't know if we should expand the command to multiple files, let's say we have
`kind load image file1 file2 file3`  and file2 fails.

The command exits with error , but file1 was already loaded.

I think that is safer to check that only one argument is passed

@BenTheElder WDYT?

Reply:
> The command exits with error , but file1 was already loaded.

This can be indicated clearly in the error message though. Or we can add a `--ignore-error` kind of an argument and turn the failed load into a warning and continue to load the rest of the archive. 

Having multiple archive load can be really useful though

Reply:
> Having multiple archive load can be really useful though

That's less clear actually, multiple invocations can be made at the same performance and you can instead create a single tarball with all your images at greater efficiency than any of these.

Reply:
> a single tarball with all your images at greater efficiency than any of these

This is very much valid. But at the same time if you are loading up a bunch of archives collected from different places (Yes, we do this internally 😢 in our dev workflows),  it is not going to be easy to aggregate them into one common tar ball without doing a few things first. 

So shall I turn this PR to enforce the `Nargs=1` behavior instead then or will that be considered a backward incompatible changes of sort ?

Reply:
> multiple invocations can be made at the same performance 

True. But with support for multiple one, you can do `kind load image-archive *.tar --name test`. Would this not be a bit more user friendly ?

Reply:
I think that first we have to validate that all files exist, and then try to load all ... basically is split this in 2 loops

Reply:
@aojea Done. PTAL.

Reply:
> I think it's OK to implement this but I almost think we should throw a warning or something to nudge people in the right direction,

@aojea @BenTheElder  Took care of this one too by adding `PreRun` hook.